apps/mark/frontend/src/api.ts

```diff
@@ -122,6 +123,40 @@ export const checkPin = {
   },
 } as const;
 
+/* istanbul ignore next */
```

Comment:
More tests I'm punting to a follow-up. Using `istanbul ignore` comments in this commit so that we can still keep VxMark at 100% code coverage and avoid coverage regressions elsewhere